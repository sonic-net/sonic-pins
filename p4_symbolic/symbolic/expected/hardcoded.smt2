; 
(set-info :status unknown)
(declare-fun standard_metadata.ingress_port () (_ BitVec 9))
(declare-fun standard_metadata.egress_spec () (_ BitVec 9))
(assert
 (let (($x54 (= standard_metadata.ingress_port (_ bv1 9))))
 (or (or false (= standard_metadata.ingress_port (_ bv0 9))) $x54)))
(assert
 (let ((?x34 (concat (_ bv0 8) (_ bv0 1))))
 (let (($x35 (= standard_metadata.ingress_port ?x34)))
 (let (($x37 (and true $x35)))
 (let (($x38 (and true (not $x35))))
 (let ((?x43 (ite $x38 ?x34 (ite $x37 (concat (_ bv0 8) (_ bv1 1)) standard_metadata.egress_spec))))
 (or (or (= ?x43 (_ bv455 9)) (= ?x43 (_ bv0 9))) (= ?x43 (_ bv1 9)))))))))
(assert
 (let ((?x34 (concat (_ bv0 8) (_ bv0 1))))
 (let (($x35 (= standard_metadata.ingress_port ?x34)))
 (let (($x37 (and true $x35)))
 (let (($x44 (ite $x35 $x37 false)))
 (let (($x38 (and true (not $x35))))
 (let ((?x43 (ite $x38 ?x34 (ite $x37 (concat (_ bv0 8) (_ bv1 1)) standard_metadata.egress_spec))))
 (let (($x47 (= ?x43 (_ bv455 9))))
 (and (and (not $x47) $x44) (= (- 1) (- 1)))))))))))
(check-sat)

; 
(set-info :status unknown)
(declare-fun standard_metadata.ingress_port () (_ BitVec 9))
(declare-fun standard_metadata.egress_spec () (_ BitVec 9))
(assert
 (let (($x54 (= standard_metadata.ingress_port (_ bv1 9))))
 (or (or false (= standard_metadata.ingress_port (_ bv0 9))) $x54)))
(assert
 (let ((?x34 (concat (_ bv0 8) (_ bv0 1))))
 (let (($x35 (= standard_metadata.ingress_port ?x34)))
 (let (($x37 (and true $x35)))
 (let (($x38 (and true (not $x35))))
 (let ((?x43 (ite $x38 ?x34 (ite $x37 (concat (_ bv0 8) (_ bv1 1)) standard_metadata.egress_spec))))
 (or (or (= ?x43 (_ bv455 9)) (= ?x43 (_ bv0 9))) (= ?x43 (_ bv1 9)))))))))
(assert
 (let (($x38 (and true (not (= standard_metadata.ingress_port (concat (_ bv0 8) (_ bv0 1)))))))
 (let ((?x34 (concat (_ bv0 8) (_ bv0 1))))
 (let (($x35 (= standard_metadata.ingress_port ?x34)))
 (let (($x45 (ite $x35 false $x38)))
 (let (($x37 (and true $x35)))
 (let ((?x43 (ite $x38 ?x34 (ite $x37 (concat (_ bv0 8) (_ bv1 1)) standard_metadata.egress_spec))))
 (let (($x47 (= ?x43 (_ bv455 9))))
 (and (and (not $x47) $x45) (= (- 1) (- 1)))))))))))
(check-sat)

