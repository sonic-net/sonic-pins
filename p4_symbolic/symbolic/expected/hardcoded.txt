Finding packet for table tbl_hardcoded55 and row -1
	Dropped = 0
	standard_metadata.ingress_port = #b000000000
	standard_metadata.egress_spec = #b000000001

Finding packet for table tbl_hardcoded57 and row -1
	Dropped = 0
	standard_metadata.ingress_port = #b000000001
	standard_metadata.egress_spec = #b000000000

